### PR TITLE
Projector config at root becomes default config for any present runs

### PR DESCRIPTION
**Pull request by hfiller**
Update projector config to better handle multiple runs. If multiple runs are present, only one set of config/metadata needs to be present in the root directory, which will be used for any runs without config explicitly set.

### CODE REVIEW

File: tensorboard/plugins/projector/projector_plugin.py [hunk 1]
```diff
@@ -422,6 +431,14 @@ def _read_latest_config_files(self, run_path_pairs):
                 with tf.io.gfile.GFile(config_fpath, "r") as f:
                     file_content = f.read()
                 text_format.Merge(file_content, config)
+            elif tf.io.gfile.exists(default_config_fpath):
+                with tf.io.gfile.GFile(default_config_fpath, "r") as f:
```
Comment (nfelt) [inline tensorboard/plugins/projector/projector_plugin.py:435]: This looks like it would read the same file contents over and over, once for each run that doesn't have its own projector config, which seems unnecessary. Can we restructure so that we only read the file at most once?  This isn't such a big deal for local filesystems but is more of a downside for e.g. GCS log directories.

File: tensorboard/plugins/projector/projector_plugin.py [hunk 2]
```diff
@@ -422,6 +431,14 @@ def _read_latest_config_files(self, run_path_pairs):
...
+                    file_content = f.read()
+                text_format.Merge(file_content, config)
+                for embedding in config.embeddings:
```
Comment (nfelt) [inline tensorboard/plugins/projector/projector_plugin.py:438]: I'm a little unclear about why this branch (the default root config) needs to expand embedding metadata paths to absolute here, whereas the other branch (the per-run config) doesn't need to do this.  Can you add some brief commentary about this?  E.g. is it because the paths are interpreted relative to the run path, or something like that?

File: tensorboard/plugins/projector/projector_plugin.py [hunk 3]
```diff
@@ -387,6 +387,10 @@ def _augment_configs_with_checkpoint_info(self):
             for tensor_name, tensor_shape in var_map.items():
                 if len(tensor_shape) != 2:
                     continue
+                # Optimizer slot values are the same shape as embeddings
+                # but are not embeddings.
+                if ".OPTIMIZER_SLOT" in tensor_name:
```
Comment (nfelt) [inline tensorboard/plugins/projector/projector_plugin.py:392]: I realize this is a small tweak, but since it doesn't appear to be related to the main change, and it's a functional change (as opposed to just refactoring or formatting), it would be ideal to break this out as its own PR.

File: tensorboard/plugins/projector/projector_plugin.py (hunk 2 above)
Comment (hfiller) [reply tensorboard/plugins/projector/projector_plugin.py:438]: That's exactly it. It's interpreted relative to the run path. I'll add a comment explaining. :)

File: tensorboard/plugins/projector/projector_plugin.py (hunk 3 above)
Comment (hfiller) [reply tensorboard/plugins/projector/projector_plugin.py:392]: 👍

Comment (hfiller) [reply tensorboard/plugins/projector/projector_plugin.py]: #3679
